README.md

```diff
@@ -4,12 +4,31 @@ Notation is a project to add signatures as standard items in the registry ecosys
 
 ## Table of Contents
 
+- [Notation Quick Start](#notation-quick-start)
 - [Branch](#branch)
 - [Core Documents](#core-documents)
 - [Community](#community)
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
 
+## Notation Quick Start
+
+- Install the Notation CLI from [Notation Releases][notation-releases]  
+  ```bash
+  curl -Lo notation.tar.gz https://github.com/shizhMSFT/notation/releases/download/v0.5.2/notation_0.5.2_linux_amd64.tar.gz
+  tar xvzf notation.tar.gz -C ~/bin notation
+  ```
+- Build, Push, Sign, Verify the `net-monitor` software
+  ```bash
+  export IMAGE=localhost:5000/net-monitor:v1
+  docker build -t $IMAGE https://github.com/wabbit-networks/net-monitor.git#main
+  docker push $IMAGE
+  notation cert generate-test --default --trust "wabbit-networks-dev"
+  notation sign $IMAGE
+  notation list $IMAGE 
```

Comment:
A comments explaining what this command does will help.

Reply:
Similar to above and #78 
How would you suggest the quick-start balance, compared to longer explanations?

Once we get automated builds, users could test this themselves, and call `--help` to get more details or see #78

Reply:
```suggestion
  # List the image, and any associated signatures
  notation list $IMAGE 
```